src/js/modules/infragistics.ui.editors.js

```diff
@@ -12059,5 +12062,695 @@
 			}
 		}
 	});
+	$.widget("ui.igTimePicker", $.ui.igDateEditor, {
+		options: {
+			/* type="object" Gets/Sets delta-value which is used to generate the drop-down items for the time picker.
+				Accepted values for deltas are positive integer numbers, and the fractional portion of floating point numbers is ignored.
+			```
+				//Initialize with object
+				$(".selector").%%WidgetName%%({
+					itemsDelta: {
+						hours: 0,
+						minutes: 30,
+					}
+				});
+
+				//Get
+				var itemsDelta= $(".selector").%%WidgetName%%("option", "itemsDelta");
+
+				//Set with object
+				$(".selector").%%WidgetName%%("option", "itemsDelta", { minutes: 30 });
+			```
+			object type="object" A configuration object, which defines specific values for each time period. The option can accept the following format:
+				itemsDelta: {
+					hours: 0,
+					minutes: 30,
+				}
+			Time periods that don't have values use 0 as default for hours and 30 for minutes.
+			*/
+			itemsDelta: { hours: 0, minutes: 30 },
+			/* type="number|object" Gets/Sets delta-value which is used to increment or decrement the editor date on spin actions.
+				When not editing (focused) the delta is applied on the day if available in the input mask or the lowest available period.
+				When in edit mode the time period, where the cursor is positioned, is incremented or decremented with the defined delta value.
+				Accepted values for deltas are positive integer numbers, and the fractional portion of floating point numbers is ignored.
+			```
+				//Initialize with number
+				$(".selector").%%WidgetName%%({
+					spinDelta: 10
+				});
+				//Initialize with object
+				$(".selector").%%WidgetName%%({
+					spinDelta: {
+						hours: 12,
+						minutes: 15
+					}
+				});
+
+				//Get
+				var spinDelta= $(".selector").%%WidgetName%%("option", "spinDelta");
+
+				//Set with number
+				$(".selector").%%WidgetName%%("option", "spinDelta", 10);
+				//Set with object
+				$(".selector").%%WidgetName%%("option", "spinDelta", { minutes: 15 });
+			```
+			number type="number" This value it is applied to hours and minutes,.
+			object type="object" A configuration object, which defines specific values for each time period. The option can accept the following format:
+				spinDelta: {
+					hours: 12,
+					minutes: 15
+				}
+			Default value is {hours: 1, minutes: 30}.
+			*/
+			spinDelta: { hours: 1, minutes: 30 },

```

Comment:
I know the override is to hide the date parts, but the inherited [`spinDelta`](https://www.igniteui.com/help/api/2017.2/ui.igdateeditor#options:spinDelta) also has seconds and milliseconds. If those still valid in the time picker mask, t maybe the delta options should stay too.

Reply:
I've talked with Slav and we agreed to have only hours and minutes

Reply:
Yeah, ignore that one - noticed the s/ms are not part of the input format.